routes/api.php

```diff
@@ -13,7 +14,8 @@
     });
     /* Routes related to the User API */
     Route::group(['prefix' => 'user', 'as' => 'user::'], static function () {
-        Route::group(['middleware' => ['auth:api'], 'as' => 'qr::'], static function () {
+        Route::get('info', fn () => Auth::user())->name('info')->middleware('auth:api');
```

Comment:
Why is this here?

Reply:
This was first in the userapi controller, but that has been removed. It is used to check if the login was succesfull.